dvc/dagascii.py

```diff
@@ -264,7 +266,7 @@ def _build_sugiyama_layout(vertexes, edges):
     # Y
     #
 
-    vertexes = {v: Vertex(" {} ".format(v)) for v in vertexes}
+    vertexes = {v: Vertex(" {} ".format(decode(v, "utf-8"))) for v in vertexes}
```

Comment:
This feels like you are treating symptoms not cause.

Reply:
Agree on this one